Taeryoung/app.js

```diff
@@ -0,0 +1,30 @@
+const http = require ("http");
+const express = require ("express");
+const cors = require ("cors");
+const morgan = require ("morgan");
+const dotenv = require("dotenv");
+dotenv.config()
+
+app = express()
```

Comment:
상수 변수를 선언하기 위해 const 키워드를 붙여주세요!
```suggestion
const app = express()
```